app/src/main/res/values/strings.xml

```diff
@@ -105,6 +105,25 @@ https://github.com/RocketChat/java-code-styles/blob/master/CODING_STYLE.md#strin
         <item>Chinese (TW)</item>
     </string-array>
 
+    <string-array name="locales">
```

Comment:
Let's keep it as a const as discussed above @HusseinElFeky.
Could you do that?

Reply:
I tried declaring the array as `private const val LOCALES = arrayOf(<locales>)` on the top side of the class, but unfortunately couldn't because `arrayOf(<elements>)` is not a primitive in Kotlin. :/ It has to be declared as `private val LOCALES = arrayOf(<locales>)`. Is that OK? or is there a workaround? And if that's OK, shall I keep all the array elements as `String` and use a comma as a delimiter, or revert back to using `String` with `Pair` together? Which is better?

Reply:
@HusseinElFeky I've updated it*. Check it out.
Thank you very much for being active here and sharing with me interesting readings and being open-minded (just want to thank you again - now here). 👊 

* Since it is not a `const` there is no reason to keep it at the top the class.

Reply:
Yup, now I get it. You are very welcome! :fist_oncoming: Glad to help. :smiley: